desktop/analytics/partner.js

```diff
@@ -22,4 +22,20 @@
       partner_slug: $(e.currentTarget).data('partner-slug')
     })
   })
+
+  analyticsHooks.on('renderedPartner2ContactInfo', function (options) {
```

Comment:
Oh sorry, I forgot to write about this. The entire contact page is loaded asynchronously, and when the tracking code is defined directly within `(function () { ... }`, jQuery fails to find the link because it's not there in the DOM. In order to attach an event properly, we need to make sure jQuery attempts to attach an event after the contact page is ready.